app/controllers/security_group_controller.rb

```diff
@@ -14,8 +14,6 @@ def self.display_methods
     %w(instances network_ports)
   end
 
-  menu_section :net
-
```

Comment:
Thanks for the pointer on remember_tab_url!  It looks like this controller already has a section there in euwe, so things should be set.